.test-infra/kafka/strimzi/02-kafka-persistent/README.md

```diff
@@ -25,4 +25,19 @@ https://github.com/strimzi/strimzi-kafka-operator/blob/main/examples/kafka/kafka
 The directories within [base](./base) contain the operator manifests and
 are named according to the version release.
 
-See [.test-infra/kafka/strimzi](../README.md) for requirements and usage.
+# Usage
+
+Simply deploy the cluster by using kustomize plugin of kubectl
+```
+kubectl apply -k .test-infra/kafka/strimzi/02-kafka-persistent
+```
+and wait until the cluster is deployed
+```
+kubectl wait kafka beam-testing-cluster --for=condition=Ready
```

Comment:
I kept getting a timeout. I didn't have time to investigate this. Either investigate this or recommend using https://k9scli.io/

Reply:
Added a timeout. Value of 1200 seems long but there are cases when deployment takes longer due to how Autopilot scales.